WatchySim/WatchFaces/AnalogGabel/Watchy_AnalogGabel.h

```diff
@@ -2,9 +2,13 @@
 #define WATCHY_ANALOGGABELT_H
 
 #ifdef WATCHY_SIM
-#include "..\..\Watchy.h"
+    #ifdef _WIN32
```

Comment:
I'm really reluctant to have the code for watch faces (the users' pride and joy) have this much evidence of the watchysim implementation. I felt dirty enough that they have to have even one ifdef.

Reading below, I didn't see a reason for there to be an entirely new file for Watch_SDL2. Can we put whatever branching we need inside of watchy.h?